src/actions/usedCountryActions.js

```diff
@@ -0,0 +1,6 @@
+export const updateUsedCountries = (country) => {
```

Comment:
We will also need to add functionality to clear out the used countries array. Maybe this happens on logout?